### PR TITLE
feat(ui): display latest version in package overview

### LINKED ISSUE #330: Display installed version of a package in the UI
**Is your feature request related to a problem? Please describe.**
It is not possible to determine which package is currently installed via the UI.

**Describe the solution you'd like**
- Similar to the `list`  command the installed version of a package should be displayed on the overview and detail page.
- On the detail page also display the latestVersion.
- If an update is available display the update path e.g. `v0.0.1` -> `v0.0.2`

**Describe alternatives you've considered**
Also always display the latest version.

### PR DESCRIPTION
**Pull request by hanshal101**
Closes #330
Fixes #330

## 📑 Description
This shows the LatestVersion and the Current version (if installed) on the package overview page
Images :
- When a Package is installed 
image
- When a Package is not installed 
image

## ✅ Checks
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required

### CODE REVIEW

**Comment by hanshal101** [comment]
Hey @pmig can you please check this I think there is some mistake I made in my code repository and I couldn't update the entire code but the UI works for me. But I absolutely dont know how

File: internal/web/templates/pages/package.html [hunk 1]
```diff
@@ -72,6 +72,16 @@ <h2 class="text-reset">Package Details</h2>
             {{ .Manifest.LongDescription }}
           {{ end }}
         </span>
+        <br />
+        <span>
+          <i>
+            Latest Verrsion: {{ .LatestVersion }}
```
Comment (christophenne) [inline internal/web/templates/pages/package.html:78]: please correct the typo: Verrsion should be Version

File: internal/web/templates/pages/package.html [hunk 2]
```diff
@@ -72,6 +72,16 @@ <h2 class="text-reset">Package Details</h2>
...
+          </i>
+          <br />
+          {{ if ne .Status nil }}
```
Comment (christophenne) [inline internal/web/templates/pages/package.html:81]: why are you checking whether `.Status` is not nil? inside the `if` block the `.Status` is not being accessed at all. or am i missing something?

File: internal/web/components/pkg_detail_btns/controller.go [hunk 3]
```diff
@@ -23,7 +23,13 @@ func getId(pkgName string) string {
 	return fmt.Sprintf("%v-%v", TemplateId, pkgName)
 }
 
-func Render(w io.Writer, tmpl *template.Template, pkg *v1alpha1.Package, status *client.PackageStatus, manifest *v1alpha1.PackageManifest, updateAvailable bool) error {
```
Comment (christophenne) [inline internal/web/components/pkg_detail_btns/controller.go:26]: Could you please revert this change and check if the pipelines still pass? Because if the previous version (i.e. the long line) did not cause any errors in the pipeline, then I think you can leave it in the long version.

**Review by christophenne — CHANGES_REQUESTED** [review]
Thanks for the PR! I have two comments about it and a question, please take a look at it.

Oh and I saw there are merge conflicts too, so please rebase your branch on the latest main.

**Comment by hanshal101** [comment]
@christophenne I have rebased the branch to main but still shows conflicts. I wonder why

**Comment by christophenne** [comment]
It's possible that in the meantime something has changed in this file, and you need to rebase again. 

Can you do us another favor please: We like to squash our commits before merging them into main, such that we don't have multiple commits with the same message but only one. Please read this: [Squashing](https://www.baeldung.com/ops/git-squash-commits). Afterwards you need to force-push your branch, and it will show up in the PR as one commit. 
We didn't require this for first-time or newer contributors yet because we can also do it for them, but since you open PRs quite frequently, we kindly ask you to try it – that would help us a lot :)

**Comment by pmig** [comment]
Hi @hanshal101 @christophenne will take care about the code review :-)

**Comment by hanshal101** [comment]
Thanks @pmig I'm totally confused why my PRs are working like this.
